add-ons/fabric8/README.md

```diff
@@ -0,0 +1,50 @@
+# Fabric8 Addon
+
+This addon provides an easy way to install fabric8 on MiniShift. 
+
+Fabric8 provides a complete developer platform for creating projects, editing them and performing full CI / CD for the entire developer lifecycle.
+
+## Setup GitHub OAuth client ID and secret
+
+Fabric8 has GitHub integration letting you browse repositories, create new repositories, edit projects and setup automated CI / CD jobs with webhooks on github. 
+
+This requires an [OAuth application to be setup on your github account](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/) for fabric8 and you need to obtain the client ID and secret for the OAuth application.
```

Comment:
folks could also put in a dummy value for the redirect URI and then update it again later after they've got fabric8 running; thats just a bit more complex really; so the simplest thing that could work is
* start minishift so we can find its IP
* setup the GitHub OAuth app and env vars 
* apply the fabric8 addon